.pre-commit-config.yaml.jinja

```diff
@@ -52,7 +52,7 @@ repos:
         language: fail
         files: "\\.rej$"
   - repo: https://github.com/oca/maintainer-tools
-    rev: 2c66e72
+    rev: fd75d80
```

Comment:
Wouldn't it be better to do tag releases like everyone else? :thinking: 

That would allow autoupdating it.

Reply:
We could do maintainer-tools tag, yes.